packages/api-v2/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     "strictBindCallApply": false,
     "forceConsistentCasingInFileNames": false,
     "noFallthroughCasesInSwitch": false,
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "allowJs": true
```

Comment:
what is the allow JS for?

Reply:
yeah it's not needed, i re-did the fix from the jest config file instead

Reply:
oh oops but i think jest complains when there are any js files/imported js modules anywhere, and allowJS is one of the ways to fix it